src/components/Hub/ImageCard.tsx

```diff
@@ -45,6 +59,7 @@ const ImageLink = styled(Link)`
 
 export default function ImageCard({ image, index }: Props) {
   let { name, author, keywords, kind, description } = image
+  const theme = useTheme()
```

Comment:
I merged the theming object for material and emotion into one and use materials theming functions. Are we not seperating them again here?